docs/book/v5/core-features/error-reporting.md

```diff
@@ -1,5 +1,7 @@
 # Error reporting endpoint
 
+> Backward incompatibility introduced in Dotkernel API 4.1.0
```

Comment:
not relevant if was introduced in 4.x 
Only if was introduced in some 5.x release